client/src/partials/employees/employees.html

```diff
@@ -1,270 +1,449 @@
-              </select>
-            </div>
-          </div>
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.prenom.$invalid }">
+                <label class="control-label" for="bhima-employee-prenom">{{ 'EMPLOYEE.FIRST_NAME' | translate }}</label>
+                <div>
+                  <input type="text" class="form-control" name="prenom" id="bhima-employee-prenom" ng-model="EmployeeCtrl.employee.prenom" required>
+                </div>
+              </div>
 
-          <div class="form-group">
-            <label class="col-xs-3 control-label">{{ "EMPLOYEE.NB_SPOUSE" | translate }}:</label>
-            <div class="col-xs-9">
-              <input class="form-control" type="number" ng-model="EmployeeCtrl.session.employee.nb_spouse">
-            </div>
-          </div>
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.name.$invalid }">
+                <label class="control-label" for="bhima-employee-prenom">{{ 'EMPLOYEE.NAME' | translate }}</label>
+                <div>
+                  <input type="text" class="form-control" name="name" id="bhima-employee-name" ng-model="EmployeeCtrl.employee.name" required>
+                </div>
+              </div>
 
-          <div class="form-group">
-            <label class="col-xs-3 control-label">{{ "EMPLOYEE.NB_CHILDREN" | translate }}:</label>
-            <div class="col-xs-9">
-              <input class="form-control" type="number" ng-model="EmployeeCtrl.session.employee.nb_enfant">
-            </div>
-          </div>
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.postnom.$invalid }">
+                <label class="control-label" for="bhima-employee-prenom">{{ 'EMPLOYEE.LAST_NAME' | translate }}</label>
+                <div>
+                  <input type="text" class="form-control" name="postnom" id="bhima-employee-postnom" ng-model="EmployeeCtrl.employee.postnom" required>
+                </div>
+              </div>
 
-          <div class="form-group" >
-            <label for="employee-dob" class="col-xs-3 control-label">{{ "EMPLOYEE.LOCKED" | translate }}</label>
-            <div class="col-xs-9">
-              <input type="checkbox" ng-model="EmployeeCtrl.session.employee.locked"  ng-true-value="1" ng-false-value="0" ng-checked="EmployeeCtrl.session.employee.locked">
-            </div>
-          </div>
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.dob.$invalid }">
+                <label class="control-label" for="bhima-employee-dob">{{ 'COLUMNS.DOB' | translate }}</label>
+                <div>
+                  <input type="date" class="form-control" name="dob" id="bhima-employee-dob" ng-model="EmployeeCtrl.employee.dob" min="{{ EmployeeCtrl.minDOB | date : 'yyyy-MM-dd'}}" max="{{ EmployeeCtrl.maxDOB | date : 'yyyy-MM-dd'}}" required>
```

Comment:
Since @DedrickEnc's date editor component was merged this weekend, could you use it for the date of birth input?  And any other date inputs on this page.  There is some documentation on how to use it in the [pull request](https://github.com/IMA-WorldHealth/bhima-2.X/pull/102).